lib/Monitor/Elembio/Staging.pm

```diff
@@ -0,0 +1,131 @@
+package Monitor::Elembio::Staging;
+
+use Carp;
+use Cwd 'abs_path';
+use File::Basename;
+use Readonly;
+use File::Spec::Functions 'catfile';
+use Exporter;
+use Perl6::Slurp;
+use JSON;
+
+our @ISA= qw( Exporter );
+our @EXPORT = qw( find_run_folders );
+
+use Monitor::Elembio::Enum qw( 
+  $RUN_MANIFEST_FILE
+  $RUN_PARAM_FILE
+  $RUN_STANDARD
+  $RUN_TYPE
+);
+Readonly::Scalar my $STAGING_GLOB => 'AV*/**/' . $RUN_MANIFEST_FILE;
+
+our $VERSION = '0';
+
+=head2 find_run_folders
+
+Find valid run folders for Elembio runs in a top folder (or staging area).
```

Comment:
Add 'folders for non-sequencing runs are excluded'. Also, add whatis returned, ie 'A list of run folder paths is returned'